### ISSUE TITLE
Raisinbread instruments should not be in PHP

### ISSUE THREAD

**Issue by driusan** [aces/Loris#8444]
Providing examples of instruments in PHP where it's not required sets a bad example and makes maintenance more difficult than it needs to be.

We should convert the ones that can be converted to LINST. For the ones that can't be converted, we should put a disclaimer at the top explaining why it can't be done in LINST and why it's provided as an example, suggesting that unless you're doing the same thing that can't be done in LINST you should use the instrument builder instead.

**Comment by ridz1208** [comment]
@driusan I'm fine with the disclaimer but until we can do everything with LINST that we can do in PHP. projects will still require PHP and I prefer pointing them to examples instead of having to spell out every possible function they can use. That being said. yes those instruments need a ton of TLC

**Comment by driusan** [comment]
Yes, that's why I say the ones that can be (just radiology_review?) should be converted. The ones that can't be should have a disclaimer explaining what they're an example of that can't be done in LINST.. the goal being to make people question if they require using PHP with all the maintenance overhead it requires when basing an instrument on one of them.

**Comment by ridz1208** [comment]
yeah that sounds okai. Also not sure if you read the docs for instrument building in the new wiki but you might want to add a disclaimer there as well ?
